packages/installations/src/functions/get-token.test.ts

```diff
@@ -96,10 +96,10 @@ const setupInstallationEntryMap: Map<
         }
       };
 
-      set(appConfig, entry);
+      await set(appConfig, entry);
 
-      // Finish pending request in 10 ms
-      sleep(50).then(() => {
+      // Finish pending request after 500 ms
+      sleep(500).then(async () => {
```

Comment:
why not await this sleep instead of then()ning it?

Reply:
ok, I see :P

you don't need the async/await if you're deliberately not doing anything with the Promise that set() returns, though :stuck_out_tongue: